packages/voila/src/manager.ts

```diff
@@ -194,7 +194,125 @@ export class WidgetManager extends JupyterLabManager {
     });
   }
 
+  /**
+   * This is the implementation of building widgets models making use of the
+   * jupyter.widget.control comm channel
+   */
   async _build_models(): Promise<{ [key: string]: base.WidgetModel }> {
+    const models: { [key: string]: base.WidgetModel } = {};
+    const commId = base.uuid();
+    const initComm = await this._create_comm(
+      'jupyter.widget.control',
+      commId,
+      { widgets: null },
+      { version: '1.0.0' }
```

Comment:
Can we get the comm name and version from `ipywidgets` instead of hard coding it?

Reply:
This version is the version of the control channel protocol that Voila supports. If ipywidgets's control channel protocol changes to 2.0.0, we very likely need to update Voila manually and we don't want Voila to advertise that it supports 2.0.0 if it doesn't.

Reply:
Thanks for the explanation!